java-lotto/src/main/java/lotto/domain/Lottoes.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lottoes {
+    private final List<Lotto> lottoes;
+
+    public Lottoes(final int amount) {
+        lottoes = create(amount);
+    }
+
+    public Lottoes(final List<Lotto> lottoes) {
+        this.lottoes = lottoes;
+    }
+
+    private List<Lotto> create(final int amount) {
+        final List<Lotto> result = IntStream.rangeClosed(1, amount)
```

Comment:
return을 곧바로 해도 되지 않을까요?

```suggestion
        return IntStream.rangeClosed(1, amount)
```